### PR TITLE
Claim metadata

### PR DESCRIPTION
**Pull request by marcdeb1**
- Added new metadata to the claim page and removed author (#50 #27). We can now view the claim bid and effective amount.
- Matched Blocked time and Created time in case of a chainquery resync (#52)

### CODE REVIEW

**Comment by tzarebczan** [comment]
Hey @marcdeb1, thanks for this PR...I totally missed it! We'll get a review on it shortly. There will be a few more fields we'll eventually want to add from the metadata, but they aren't available in chainquery yet.

**Comment by tiger5226** [comment]
@marcdeb1 They are actually available on dev for chainquery https://dev.chainquery.lbry.com/api/sql?query=SELECT%20*%20FROM%20chainquery.claim%20WHERE%20claim_id=%22c2bea0127e2543a5664e51cc32833851cf9dfda5%22 

The commit is https://github.com/lbryio/chainquery/commit/b1dab7cfa912153038e14170856f2ae2fcf66262

you can see the sql migration to see what columns were added.

File: src/Template/Main/claims.ctp [hunk 1]
```diff
@@ -44,8 +44,9 @@ $cost = 'Free';
 if (isset($claim->price) && $claim->price > 0) {
     $cost = $this->Amount->formatCurrency($claim->price) . ' LBC';
 } else if (isset($claim->fee) && strtolower($claim->fee_currency) === 'lbc') {
-    $cost = $this->Amount->formatCurrency($claim->fee) . ' LBC';
+    $cost = (float) ($claim->fee) . ' LBC';
```
Comment (tiger5226) [inline src/Template/Main/claims.ctp]: why are we treating currency as a float? What was wrong with `formatCurrency`?

File: src/Template/Main/claims.ctp [hunk 2]
```diff
@@ -44,8 +44,9 @@ $cost = 'Free';
...
 }
+$effective_amount = $claim->effective_amount / 100000000;
```
Comment (tiger5226) [inline src/Template/Main/claims.ctp]: can we create a const instead so the number makes sense?

File: src/Template/Main/claims.ctp [hunk 3]
```diff
@@ -101,6 +106,17 @@ if (strlen(trim($desc)) == 0) {
 
             <div class="clear"></div>
             <?php endif; ?>
+            
+            <div class="half-width"><div class="label help-text" title="The quantity of tokens used to back the stake.">Claim bid</div></div>
+            <div class="half-width"><div class="label help-text" title="The sum of the claim bid and all of its active supports.">Effective amount</div></div>
+
+            <div class="value half-width"><?php echo (float) ($claim->claim_bid) . ' LBC'; ?></div>
+            <div class="value half-width"><?php echo ($effective_amount < $claim->claim_bid) ? '<em>Synchronizing</em>' : ((float) ($claim->effective_amount / 100000000) . ' LBC');  ?></div>
```
Comment (tiger5226) [inline src/Template/Main/claims.ctp]: same thing here..using the const would be best...why not just fix the format currency function?

**Comment by tzarebczan** [comment]
@marcdeb1 did you still want to finish this PR?

**Comment by tzarebczan** [comment]
Ping @marcdeb1  :)

File: src/Template/Main/claims.ctp (hunk 3 above)
Comment (marcdeb1) [reply src/Template/Main/claims.ctp]: I added a function in AmountHelper to format LBRY prices, to make it more reuseable.

**Comment by tiger5226** [comment]
I ran this locally, 
- the real time stats pages recent transactions time is still based on created_at instead of transaction time. 
- the new fields have this info type hover over that the other fields do not have. 

Other than these two things, it looks good. I did not test the apis.

**Comment by kauffj** [comment]
@tiger5226 ship it?

**Comment by tiger5226** [comment]
I looked into this tonight. The environment is a bit messy. There are 3 installations. I found the one being used by caddy, but its got the blocked endpoints branch loaded there. I will deploy this after I merge the other branch. We need to fix CI too. It would be nice if it just build a docker image too and auto deployed.

**Comment by tzarebczan** [comment]
We should try to get this in and confirm it fixes this: 
image

**Pull request closed without merging**